admin/notif-services/jira.rst

```diff
@@ -69,7 +69,7 @@ To create a Jira integration in Observability Cloud:
 
 #. (Optional) Click :strong:`Create Test Issue` to test your integration. If the integration is working, it creates a test Jira issue in the selected Jira project. After a short delay, the integration makes a comment on the same issue, stating that the alert has cleared.
 
-#. strong:`Save`.
+#. strong:`Save`. (Is this supposed to be here? Seems like this is out of place. Seems like this line could be added to step 12)
```

Comment:
@brittonjohnson I understand your question, but I think we should leave this step as it stands. The optional step after step 12 gives users the option of testing their integration before they save it.

Reply:
@jmalin-splunk Another option is to directly use the Suggest feature to made the edit yourself (maintainers can edit PRs).